### ISSUE TITLE
Show number of comments over time

### ISSUE THREAD

**Issue by collinanderson** [social-protocols/news#150]
Would be nice to see number of comments over time, similar to number of upvotes over time. I realize that there are hidden and dead comments, so it's not a perfect number, but getting a rough number over time would be pretty interesting, especially if the number of comments might affect the ranking.

**Comment by johnwarden** [comment]
I agree that could be a nice feature. To add this, we'd need to add a DB column for the number of columns, modify the crawl to write this value, and then add a chart to the stats page.
